submissions/Vitaminvp/HTML&CSSPractice/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Responsive Side Menu</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<header>
+  <nav class="nav">
+      <input type="checkbox" class="nav__input" id="nav__input">
+      <label for="nav__input">
+          <div class="nav__input_icons">
+              <div class="nav__input_icon">&#9632;</div>
+              <div class="nav__input_icon">&#9632;</div>
+              <div class="nav__input_icon">&#9632;</div>
+          </div>
+      </label>
+      <div class="nav__menu">
+          <ul class="nav__menu_list">
+              <li><a href="#"><span class="nav__menu_icon "></span> <span class="nav__menu_title">Обліковий запис</span></a></li>
```

Comment:
I noticed you used BEM in this project. But as I know, according to this methodology, each element should have `className`, but tags `li` and `a` in your code dont have them. Is not it? 
Please, let me know if I'm wrong